product/reports/670_Performance by Asset Type - Middleware Servers/160_Web Metrics - hourly over last day.yaml

```diff
@@ -0,0 +1,61 @@
+---
+where_clause:
+dims:
+created_on: 2016-10-09 14:13:30.822035 Z
+reserved:
+title: "Web Metrics - hourly over the last day"
+conditions:
+updated_on: 2016-10-09 17:23:33.732482 Z
+order: Descending
+graph:
+menu_name: "Web Metrics - hourly over the last day"
```

Comment:
is the title specific enough? it is not clear from it that it's regarding web metrics